src/test/java/jewellery/inventory/integration/SaleIntegrationTest.java

```diff
@@ -137,24 +172,6 @@ void createSaleSuccessfully() {
         saleResponse.getBody().getProducts().get(0).getId());
   }
 
-  @NotNull
-  private static SaleRequestDto getSaleRequestDto(
-      User seller, User buyer, ResponseEntity<ProductResponseDto> productResponse) {
-    SaleRequestDto saleRequestDto = new SaleRequestDto();
-    saleRequestDto.setBuyerId(buyer.getId());
-    saleRequestDto.setSellerId(seller.getId());
-    saleRequestDto.setDate(Date.from(Instant.now()));
-    ProductPriceDiscountRequestDto productPriceDiscountRequestDto =
-        new ProductPriceDiscountRequestDto();
-    productPriceDiscountRequestDto.setProductId(productResponse.getBody().getId());
-    productPriceDiscountRequestDto.setSalePrice(10000.0);
-    productPriceDiscountRequestDto.setDiscount(10.0);
-    List<ProductPriceDiscountRequestDto> list = new ArrayList<>();
-    list.add(productPriceDiscountRequestDto);
-    saleRequestDto.setProducts(list);
-    return saleRequestDto;
-  }
```

Comment:
Something strange has happened with this method. Why is it moved in the beginning of the file?

Reply:
The formatter did this... probably my formatter actually. Fixed it.